Content.MapRenderer/MapViewerData.cs

```diff
@@ -43,16 +45,19 @@ public sealed class LayerGroup
     public GroupSource Source { get; set; } = new();
     public List<Layer> Layers { get; set; } = new();
 
-    public static LayerGroup DefaultParallax()
+    public static LayerGroup DefaultParallax(IResourceManager resourceManager, ParallaxOutput output)
     {
         return new LayerGroup
         {
             Scale = new Position(0.1f, 0.1f),
             Source = new GroupSource
             {
-                Url = "https://i.imgur.com/3YO8KRd.png",
-                Extent = new Extent(6000, 4000)
+                Url = output.ReferenceResourceFile(resourceManager, new ResPath("/Textures/Parallaxes/layer1.png")),
+                Extent = new Extent(6000, 4000),
             },
+            // REMOVED because using Imgur as a CDN is utterly cringe,
+            // and I can't figure out the source files for these.
+            /*
```

Comment:
The source dmi is inside the attribution for that png.
https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Parallaxes/meta.json
https://github.com/tgstation/tgstation/blob/master/icons/effects/parallax.dmi.

Please add the other layers

Reply:
That `meta.json` isn't even correct since that isn't an RSI. FFS.